6 view all comments by article

MFStevenson opened the pull request.

MFStevenson committed:
allows uer to view comments

popatre commented on src/App.jsx:
```diff
@@ -1,18 +1,33 @@
+import CommentsPage from "./pages/CommentsPage";
 
 function App() {
+  const [articles, setArticles] = useState([]);
```
Cant see anything else thats using this state - so probably doesnt need to be in app

popatre commented on src/components/Articles.jsx:
```diff
@@ -1,10 +1,36 @@
+  };
+
+  const handleSubmit = (e) => {
+    console.log(e);
```
remove log

popatre commented on src/components/Articles.jsx:
```diff
@@ -1,10 +1,36 @@
+  const [input, setInput] = useState(0);
+  const navigate = useNavigate();
+
+  const updateInput = (e) => {
```
Would a user know the id of the article they want?

popatre commented on src/pages/CommentsPage.jsx:
```diff
@@ -0,0 +1,45 @@
+      <h2>Comments</h2>
+      <ul>
+        {comments.map((comment) => {
+          console.log(comment);
```
remove log

popatre reviewed (changes requested):
Nice work so far. Really like the loading spinner

Just one thing please:
- [ ] You should add some feedback for when an article has no comments

MFStevenson committed:
attempting to merge

MFStevenson committed:
actions feedback
removes search by id
navbar buttons now work in that filters articles by topic
should diplay message stating no comments at article

popatre reviewed (approved):
Nice one - much better 👍

nc-modou reviewed (approved):
Great implementation of the comments, and the 'article has no comments' message! Merge away.
